chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -3,38 +3,93 @@
 
 
 def _fast_hist(label_true, label_pred, n_class):
-    """Construct histogram for label evaluation.
+    # Construct histogram for label evaluation.
 
-    Returns
-        numpy.ndarray of shape (n_class, n_class)
-    """
     mask = (label_true >= 0) & (label_true < n_class)
+    # an array of shape (n_class, n_class)
     hist = np.bincount(
         n_class * label_true[mask].astype(int) +
         label_pred[mask], minlength=n_class**2).reshape(n_class, n_class)
     return hist
 
 
-def label_accuracy_score(label_true, label_pred, n_class):
-    """Returns accuracy score evaluation result.
+def eval_semantic_segmentation(label_pred, label_true, n_class):
+    """Evaluate results of semantic segmentation.
+
+    This function measures four metrics: pixel accuracy,
+    mean pixel accuracy, mean intersection over union and
+    frequency weighted intersection over union.
 
-      - overall accuracy
-      - mean accuracy
-      - mean IU
-      - fwavacc
+    The definition of these metrics are as follows, where
+    :math:`p_{ij}` is the amount of pixels of class :math:`i`
+    inferred to belong to :math:`j` and there is :math:`k` classes.
```

Comment:
Usually `p_{ij}` denotes a probability of `ij`. How about using `N_{ij}` instead of `p_{ij}`?